scalableminds/webknossos
# Improve skeleton proofreading and fix some shortcuts on MacOS

- Enables the existing (legacy) shortcuts for deleting/creating edges for orthogonal and arbitrary mode. Now, they are not restricted to legacy mode anymore.
- Improves the shortcut indicator in the status bar.
- Allows creating trees and deleting nodes in arbitrary mode with the keyboard.
- Treats the Cmd (meta) key in MacOS as a synonym to ctrl everywhere. 
- Shows the ⌥ symbol instead of "alt" and Cmd instead of Ctrl in the status bar and context menu on MacOS.

### URL of deployed dev instance (used for testing):
- https://improveskeletonproofreading.webknossos.xyz/

### Steps to test:
- ortho mode:
  - create a tree with some nodes
  - delete an edge by holding ctrl+shift while clicking on the neighbor of the active node (the statusbar should indicate that action while pressing the modifiers)
  - merge the edge again by holding ctrl+alt (on some linux setups, shift also needs to be pressed to avoid that the default handler for moving a window is triggered)
  - the above should also work in legacy mode
  - test that node selection still works correctly (legacy mode should require shift for that)
- ortho mode
  - c should create a new tree
  - delete should delete the active node
  - clicking on a node should select it (sometimes the node is then not precisely visible, I don't really know how we could circumvent this)
  - as above, try out creating and deleting edges
- on mac:
  - test that https://github.com/scalableminds/webknossos/issues/7652 is fixed
  - test that https://github.com/scalableminds/webknossos/issues/7584 is fixed

### Issues:
- https://scm.slack.com/archives/C5AKLAV0B/p1705582675454289
- fixes #7584
- fixes #7652

------
(Please delete unneeded items, merge only when none are left open)
